spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb

```diff
@@ -0,0 +1,518 @@
+FakeTerraformRepo = Spec::Support::FakeAnsibleRepo # TODO: replace FakeAnsibleRepo with FakeTerraformRepo
```

Comment:
FYI here is an example of a "pluggable" support repo class https://github.com/ManageIQ/manageiq-providers-workflows/blob/master/spec/support/fake_workflows_repo.rb

Reply:
Should be resolved